avaframe/com1DFA/com1DFACfg.ini

```diff
@@ -102,6 +102,8 @@ cMax = 0.5
 cflIterConstant = 5
 # to constrain the lower margin of the allowed time step
 constrainCFL = False
+# to use a variable time step (time step depends on kernel radius)
+moussaTimeStepping = False
```

Comment:
Mh, maybe use a more descriptive name for this instead of the authors name? Somethink like kernRadTimeStepping?